src/shared/install.c

```diff
@@ -1708,36 +1708,58 @@ static int install_info_discover_and_check(
         return install_info_may_process(ret ? *ret : NULL, paths, changes, n_changes);
 }
 
-int unit_file_verify_alias(const UnitFileInstallInfo *i, const char *dst) {
+int unit_file_verify_alias(const UnitFileInstallInfo *i, const char *dst, char **ret_dst) {
+        _cleanup_free_ char *dst_updated = NULL;
         int r;
 
-        if (unit_name_is_valid(i->name, UNIT_NAME_TEMPLATE) &&
-            !unit_name_is_valid(dst, UNIT_NAME_TEMPLATE) &&
-            !i->default_instance)
-                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                       "Aliases for templates without DefaultInstance must be templates.");
+        const char *path_alias = strrchr(dst, '/');
```

Comment:
basename?

Reply:
In this case no, because we need to check if "/" is  in the path.